server/www/classes/Telemetry.js

```diff
@@ -1,19 +1,54 @@
-class Telemetry{
-    constructor(_name, isTimeBased, unit = undefined){
-        this.name = _name;
+/*Telemetries can be of two types: 
+ 
+- "default"
+    in this case, data, pendingData, values and xy are used and textFormatValue is undefined
 
-        this.data = [[],[]]; // data[0] contains the timestamps and data[1] contains the values corresponding to each timestamp
-        this.pendingData = [[],[]];
+- "textBased"
+    in this case, data, pendingData, values and xy are not used and textFormatValue is used
+  
+ */
 
-        this.unit = ( unit != "" ) ? unit : undefined;
-        this.xy = !isTimeBased;
 
-        this.flags = undefined;
+class Telemetry{
+    constructor(_name, isTimeBased, unit = undefined, type = "default"){
+        this.type = type; // either "default" or "textBased"
```

Comment:
`number` and `text` seems like good namings for the types (this is is really a detail)